upgrade-guide.md

```diff
@@ -0,0 +1,12 @@
+# AWS Platform Features Upgrade Guide
+This guide helps in migration from the old terraform modules to the new one.
+
+# Updgrade guide to AWS platform features module from 0.2.x to 0.3.x
+1. When upgrading terraform version for platform features ensure that you are running on version `0.2.x` and the platform features is upgraded to the newer 0.3.x version.
```

Comment:
Here you have to list down the latest version in `0.2.x` and similarly for `0.3.x`